### PR TITLE
Add tooltip for remove, up and down button on MaterialArrayLayout and MaterialArrayControl 

### PR DESCRIPTION
**Pull request by Maxouwell**
PR for #2284

### CODE REVIEW

File: packages/material-renderers/test/renderers/MaterialArrayLayout.test.tsx [hunk 1]
```diff
@@ -629,4 +634,141 @@ describe('Material array layout', () => {
...
 });
+
+function checkTooltip(
+  wrapper: ReactWrapper<any, any>,
+  wrapperFindFunction: (
+    wrapper: ReactWrapper<any, any>
+  ) => ReactWrapper<any, any>,
+  tooltipEnum: ArrayTranslationEnum,
+  tooltipFilter: EnzymePropSelector | string
+) {
```
Comment (lucas-koehler) [inline packages/material-renderers/test/renderers/MaterialArrayLayout.test.tsx:726]: Please type the return type of this function explicitly.

File: packages/material-renderers/test/renderers/MaterialArrayLayout.test.tsx [hunk 2]
```diff
@@ -629,4 +634,141 @@ describe('Material array layout', () => {
...
+  return wrapper;
+}
+
+function checkTooltipTranslation(
+  wrapper: ReactWrapper<any, any>,
+  wrapperFindFunction: (
+    wrapper: ReactWrapper<any, any>
+  ) => ReactWrapper<any, any>,
+  tooltipFilter: EnzymePropSelector | string
+) {
```
Comment (lucas-koehler) [inline packages/material-renderers/test/renderers/MaterialArrayLayout.test.tsx:752]: Please type the return type of this function explicitly.

File: packages/material-renderers/test/renderers/MaterialArrayLayout.test.tsx [hunk 3]
```diff
@@ -629,4 +634,141 @@ describe('Material array layout', () => {
...
+      {
+        id: 'tooltip-down',
+      }
+    );
+  });
 });
+
+function checkTooltip(
+  wrapper: ReactWrapper<any, any>,
+  wrapperFindFunction: (
```
Comment (lucas-koehler) [inline packages/material-renderers/test/renderers/MaterialArrayLayout.test.tsx:721]: Can we find a more self-explanatory name for this parameter. What exactly does it find? It seems to find the Tooltip container. If you agree, we could rename it to `findTooltipContainer` or `findTooltipWrapper`. What do you think? Same for the `checkTooltipTranslation` function below

**Review by lucas-koehler — CHANGES_REQUESTED** [review]
Hi @Maxouwell ,
thank you for this contribution :heart: 
The code looks already pretty good to me and works as expected in the example app :)
I only have a few minor remarks inline. Please have a look.

**Comment by Maxouwell** [comment]
Sorry, didn't see you commented on the PR !
I added the tooltip to the MaterialArrayControl to be coherent (Exemple site was nagging me).

I doing your comments

File: packages/material-renderers/test/renderers/MaterialArrayLayout.test.tsx (hunk 1 above)
Comment (Maxouwell) [reply packages/material-renderers/test/renderers/MaterialArrayLayout.test.tsx]: Done

File: packages/material-renderers/test/renderers/MaterialArrayLayout.test.tsx (hunk 2 above)
Comment (Maxouwell) [reply packages/material-renderers/test/renderers/MaterialArrayLayout.test.tsx]: Done

File: packages/material-renderers/test/renderers/MaterialArrayLayout.test.tsx (hunk 3 above)
Comment (Maxouwell) [reply packages/material-renderers/test/renderers/MaterialArrayLayout.test.tsx]: and done

**Comment by Maxouwell** [comment]
(should I be the one resolving the conversations ?)

**Review by lucas-koehler — APPROVED** [review]
Hi @Maxouwell , thanks for the updates and also updating the table control :)
You can resolve comments that you deem fixed in the future but I did it for now.

**Pull request merged**